front/src/components/App/HelpArea/HelpArea.tsx

```diff
@@ -0,0 +1,31 @@
+import Button from "../../UI/Button/Button";
+import RouteLink from "../../UI/RouteLink/RouteLink.styled";
+import { APP_PAGE } from "../App";
+import { HelpAreaStyled, HelpAreaNavStyled } from "./HelpArea.styled";
+import HelpAreaInput from "./HelpArea.input";
+import { HelpAreaInputStyled } from "./HelpArea.input.styled";
+import logo from '../../../assets/logo.svg';
+
+const HelpArea = () => {
+  return (
+    <HelpAreaStyled>
+
+      <HelpAreaNavStyled>
+        <img src={logo} alt = "logo"></img>
```

Comment:
To nie powinno się tutaj znaleźć. Po to był stworzony komponent Header żeby go reużyć w takim przypadku. Teraz mamy duplikację kodu